[2025-07-13T06:00:26.487+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_etl_pipeline.load_processed_covid_data manual__2025-07-13T03:37:50.399182+00:00 [queued]>
[2025-07-13T06:00:26.515+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_etl_pipeline.load_processed_covid_data manual__2025-07-13T03:37:50.399182+00:00 [queued]>
[2025-07-13T06:00:26.517+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-07-13T06:00:26.688+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_processed_covid_data> on 2025-07-13 03:37:50.399182+00:00
[2025-07-13T06:00:26.733+0000] {standard_task_runner.py:60} INFO - Started process 781 to run task
[2025-07-13T06:00:26.760+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'main_etl_pipeline', 'load_processed_covid_data', 'manual__2025-07-13T03:37:50.399182+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/main_pipeline.py', '--cfg-path', '/tmp/tmpipsboyjc']
[2025-07-13T06:00:26.780+0000] {standard_task_runner.py:88} INFO - Job 53: Subtask load_processed_covid_data
[2025-07-13T06:00:27.192+0000] {task_command.py:423} INFO - Running <TaskInstance: main_etl_pipeline.load_processed_covid_data manual__2025-07-13T03:37:50.399182+00:00 [running]> on host 23913323947a
[2025-07-13T06:00:27.958+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_processed_covid_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-13T03:37:50.399182+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-13T03:37:50.399182+00:00'
[2025-07-13T06:00:28.141+0000] {logging_mixin.py:188} INFO - DEBUG: Contenido de processed_data recibido para carga en 'processed_covid_data': {'date': '2023-03-09T00:00:00', 'last_update_utc': '2023-03-10T04:21:03', 'total_confirmed': 7483444, 'confirmed_daily_change': 8225, 'total_deaths': 333188, 'deaths_daily_change': 52, 'total_recovered': 0, 'recovered_daily_change': 0, 'total_active': 7150256, 'active_daily_change': 8173, 'fatality_rate': 0.0445, 'country_iso': 'MEX', 'is_high_deaths_day': False, 'active_to_confirmed_ratio': 0.9554766495212632, 'recovery_rate': 0.0, 'transformation_timestamp': '2025-07-13T06:00:21.572269'}
[2025-07-13T06:00:28.158+0000] {logging_mixin.py:188} INFO - DEBUG: Tipo de processed_data: <class 'dict'>
[2025-07-13T06:00:28.170+0000] {logging_mixin.py:188} INFO - Iniciando carga de datos procesados en MongoDB en la colecci√≥n: processed_covid_data...
[2025-07-13T06:00:32.100+0000] {logging_mixin.py:188} INFO - Datos cargados exitosamente en MongoDB en covid_db.processed_covid_data.
[2025-07-13T06:00:32.102+0000] {logging_mixin.py:188} INFO - Registros cargados (procesados): 1 en processed_covid_data
[2025-07-13T06:00:32.179+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-13T06:00:32.231+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=main_etl_pipeline, task_id=load_processed_covid_data, execution_date=20250713T033750, start_date=20250713T060026, end_date=20250713T060032
[2025-07-13T06:00:32.357+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-13T06:00:32.478+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
